.github/workflows/cc-payload-after-push-amd64.yaml

```diff
@@ -20,6 +20,7 @@ jobs:
           - cc-sev-kernel
           - cc-sev-ovmf
           - cc-x86_64-ovmf
+          - cc-snp-qemu
```

Comment:
This will make `snp-qemu` part of the payload generated after every push, and I think this is **also** something that you want.

However, don't you want to also add this as part of the release?  If so, https://github.com/kata-containers/kata-containers/blob/CCv0/.github/workflows/cc-payload-amd64.yaml is where you should be looking at.

Reply:
Oh, I hadn't seen there was a separate release workflow, so I will add that to this PR for qemu, and will also add the x86_64 ovmf build that SNP needs